src/legacy/amos/zbesh.f

```diff
@@ -169,7 +169,8 @@ SUBROUTINE ZBESH(ZR, ZI, FNU, KODE, M, N, CYR, CYI, NZ, IERR)
 C***FIRST EXECUTABLE STATEMENT  ZBESH
       IERR = 0
       NZ=0
-      IF (ZR.EQ.0.0D0 .AND. ZI.EQ.0.0D0) IERR=1
+      IF ((DABS(ZR-0.0D0).LE.0.0D0) .AND. DABS(ZI-0.0D0).LE.0.0D0)
+     +IERR=1
```

Comment:
I think this syntax is incorrect. It must be on the same line.

Reply:
The idiom `ZR .EQ. 0.d0` should be translated to `ABS(ZR) .LT. EPS`. Not to `(DABS(ZR-0.0D0).LE.0.0D0`.

Reply:
There are more places where this should be done.